src/main/java/org/sonarlint/cli/SonarLint.java

```diff
@@ -61,13 +66,15 @@ public static SonarLint create(Options opts) throws IOException {
     } catch (Exception e) {
       throw new IllegalStateException("Error loading plugins", e);
     }
-    SonarLintClient c = SonarLintClient.builder()
+
+    StandaloneGlobalConfiguration config = StandaloneGlobalConfiguration.builder()
       .addPlugins(plugins)
       .setLogOutput(new DefaultLogOutput(LOGGER))
-      .setVerbose(opts.isVerbose())
+      // verbose?
```

Comment:
IIRC we dropped this parameter to always have DEBUG logs in the callback and filter on client side